### PR TITLE
REDDEV-560 vue main

### PR DESCRIPTION
**Pull request by wsams**
Restructured app to introduce Vue. I modeled this after Vizr.

Moved code from `index.php` into `lib/main.tmpl`. This is processed by webpack.

Added `ConsortReport` component. Just says hello at the moment.

I have another branch that builds upon this one that actually makes the data calls and renders the report.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,23 +25,23 @@
     "babel-eslint": "^8.2.5",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "~1.7.0",
+    "babel-preset-env": "~1.6.1",
```
Comment (kirenotneb) [inline package.json]: Why are so many dependencies downgraded?

File: js/main.js [hunk 2]
```diff
@@ -1,103 +1,31 @@
...
+    components: { ConsortReport },
+    render(createElement) {
+      return createElement(ConsortReport, {
+        props: {
+          pid: this.pid
+        }
+      });
+    },
+    data: {
+      pid
```
Comment (heathharrelson) [inline js/main.js]: Where is `pid` defined? It also looks like you're not declaring a prop for it in the `ConsortReport` component.

File: package.json (hunk 1 above)
Comment (wsams) [reply package.json]: I think I ran into some issues while working on the hello app Friday so I downgraded to the previous versions, but it appears to be working now so I'll revert them.

File: js/main.js (hunk 2 above)
Comment (wsams) [reply js/main.js]: I left this in here thinking I would need it for the subsequent work but I don't think it was needed. I removed it and the `urls` constant for now.

**Pull request closed without merging**